### DISCUSSION TITLE
Migration from ConbeeII to Sonoff 3.0 P donlge w/o repairing impossible. Correct?

### DISCUSSION THREAD

**Discussion by iz3man** [Koenkk/zigbee2mqtt#13899] [General]
Just want to make sure before I spend another afternoon googeling for a solution:

I'm running Z2M with the Conbee2 stick and want to use the Sonoff-P dongle now.
As I understand the FAQ this NEEDS manual reconnection of all my 48 devices?! Correct?

BUT; If I do so: Will Z2M correctly re-assign all my name changes etc? Or will I need to go to all of my customization again as well - which will take much more time than pressing the pairing button on all devices.

Thanks!

**Comment by Koenkk** [comment]
Repairing is needed but you can keep the names (just copy `data/configuration.yaml`). If you are also using home assistant do the following:

- stop HA
- stop Z2M
- power down all zigbee routers
- backup data/configuration.yaml (just to be sure0
- update serial section in configuration.yaml
- start z2m
- pair all devices
- start ha
